src/components/ArticleComponents/Codebox/index.tsx

```diff
@@ -20,49 +20,66 @@ const replaceLanguages = (language: string) =>
     .replace(/mjs|cjs|javascript/i, 'js')
     .replace(/console|shell/i, 'bash');
 
-const replaceLabelLanguages = (language: string) =>
-  language
-    .replace(/javascript/i, 'cjs')
-    .replace(/console|shell/i, 'bash')
-    .toUpperCase();
-
 const Codebox = ({ children: { props } }: Props): JSX.Element => {
   const [parsedCode, setParsedCode] = useState('');
   const [copied, copyText] = useCopyToClipboard();
+  const [langIndex, setLangIndex] = useState(0);
+  const [labelLang, setLabelLang] = useState('LANG');
+  const [stringCode, setStringCode] = useState('');
 
   // eslint-disable-next-line react/prop-types
-  const className = props.className || 'text';
-
+  const className = props.className || 'LANG';
   // Language Matches in class
   const matches = className.match(/language-(?<lang>.*)/);
-
   // Language name from classname
-  const language = matches?.groups?.lang || 'text';
+  const language = matches?.groups?.lang || 'LANG';
+  const LabelArray = language.split('|') || ['LANG'];
+  const codeArray = props.children
```

Comment:
I keep in mind that this piece of code can change but I like the split on this side because the switch functionality can also work for the learn section.

Reply:
Yeah, the split feels nice. Maybe if we add extra `-`, then it will be failproof 😅